ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx

```diff
@@ -1,6 +1,7 @@
+/* * @jsxRuntime classic */
```

Comment:
I can't remember if this was always set by default, so you don't need to do it here?

Reply:
Good point, think I can remove this.